doc/CORTX_on_Open_Virtual_Appliance.rst

```diff
@@ -145,21 +147,21 @@ Miscellaneous
 
 If you have a firewall between CORTX and the rest of your infrastructure, including but not limited to S3 clients, web browser, and so on, ensure that the ports mentioned below are open to provide access to CORTX.
   
-+----------------------+-------------------+---------------------------------------------+
-|    **Port number**   |   **Protocols**   |   **Destination network on CORTX**          |
-+----------------------+-------------------+---------------------------------------------+
-|          22          |        TCP        |           Management network                |
-+----------------------+-------------------+---------------------------------------------+ 
-|          53          |      TCP/UDP      | Management network and Public Data network  |
-+----------------------+-------------------+---------------------------------------------+ 
-|         123          |      TCP/UDP      |              Management network             |
-+----------------------+-------------------+---------------------------------------------+
-|         443          |       HTTPS       |             Public Data network             |
-+----------------------+-------------------+---------------------------------------------+
-|         9443         |       HTTPS       |              Public Data network            |
-+----------------------+-------------------+---------------------------------------------+
-|         28100        |   TCP (HTTPS)     |              Management network             |
-+----------------------+-------------------+---------------------------------------------+
++-----------------+---------------+--------------------------------------------+
```

Comment:
What is the real change for this table?

Reply:
I 'm not modify any, I had checked and seem like no issue, not sure why the thing pop up.